packages/genesys-spark-components/src/components/stable/gux-datepicker/tests/gux-datepicker.spec.ts

```diff
@@ -33,7 +38,31 @@ describe('gux-datepicker', () => {
           html: input,
           language
         });
+        expect(page.root).toMatchSnapshot();
+      });
+    });
+  });
 
+  describe('#render using regional dates', () => {
+    [
+      `<gux-datepicker value="1997-08-15"></gux-datepicker>`,
+      `<gux-datepicker value="1997-08-15" format="mm/dd/yy"></gux-datepicker>`,
+      `<gux-datepicker value="1997-08-15" format="mm.dd.yyyy"></gux-datepicker>`,
+      `<gux-datepicker value="1997-08-15" disabled></gux-datepicker>`,
+      `<gux-datepicker mode="range" value="2019-11-25/2019-12-02" number-of-months="2" ></gux-datepicker>`,
+      `<gux-datepicker mode="range" value="2019-11-25/2019-12-02" min-date="2019-11-10" max-date="2019-12-31" number-of-months="2" ></gux-datepicker>`,
+      `<gux-datepicker mode="range" value="2019-11-25/2019-12-02" min-date="2019-11-10" max-date="2019-12-31" number-of-months="2" format="mm.dd.yyyy" ></gux-datepicker>`,
+      `<gux-datepicker mode="range" value="2019-11-25/2019-12-02" number-of-months="2" disabled></gux-datepicker>`,
+      `<gux-datepicker value="1997-08-15" start-day-of-week="6"></gux-datepicker>`
+    ].forEach((input, index) => {
+      it(`should render component as expected (${index + 1})`, async () => {
+        // not working as expected
```

Comment:
@daragh-king-genesys @thomas-c-dillon I am trying to set the mock implementation to `true` just for this test but it is not working as expected. Any ideas? Thanks